battery.sh

```diff
@@ -520,6 +535,64 @@ if [[ "$action" == "maintain" ]]; then
 
 fi
 
+# Battery calibration
+if [[ "$action" == "calibrate_synchronous" ]]; then
+	log "Starting calibration"
+
+	# Stop the maintaining
+	battery maintain stop
+
+	# Discharge battery to 15%
+	battery discharge 15
+
+	while true; do
+		log "checking if at 100%"
+		# Check if battery level has reached 100%
+		if battery status | head -n 1 | grep -q "Battery at 100%"; then
+			break
+			else
+			sleep 300
+			continue
+		fi
+	done
+
+	# Wait before discharging to target level
+	log  "reached 100%, maintaining for 1 hour"
+	sleep 3600
+
+	# Discharge battery to 80%
+	battery discharge 80
```

Comment:
Does this make more sense than running `recover`? Is discharging to 80 part of the calibration or is it based on the assumption that we are aiming to return to a baseline of 80?

Reply:
No, the discharging is a part of the calibration.